ipfs/filestore.go

```diff
@@ -232,15 +206,15 @@ func (fs *Filestore) NewAdder(pin, wrap bool) (cafs.Adder, error) {
 	}, nil
 }
 
-// AddAndPinBytes adds a file to the top level IPFS Node
+// AddFile adds a file to the top level IPFS Node
 func (fs *Filestore) AddFile(file cafs.File, pin bool) (hash string, err error) {
 	node := fs.Node()
-
 	ctx := context.Background()
-	bserv := blockservice.New(node.Blockstore, node.Exchange)
-	dagserv := dag.NewDAGService(bserv)
 
-	fileAdder, err := coreunix.NewAdder(ctx, node.Pinning, node.Blockstore, dagserv)
+	// bserv := blockservice.New(node.Blockstore, node.Exchange)
```

Comment:
Either delete these lines, or add a TODO about what needs to be down to revive them.